jujugui/static/gui/src/app/components/inspector/inspector.js

```diff
@@ -22,18 +22,49 @@ YUI.add('inspector-component', function() {
 
   juju.components.Inspector = React.createClass({
 
+
+    /**
+      Get the current state of the inspector.
+
+      @method _getInitialState
+      @returns {String} The current state.
+    */
+    _getInitialState: function() {
+      return {
+        activeComponent: this.props.getAppState(
```

Comment:
So by assigning this as a state property you're saying that this `activeComponent` will be updated when the 'body' view of the inspector is?